src/utils/logger.js

```diff
@@ -3,6 +3,7 @@
  *
  * @author    Matt DeGennaro <mdegennaro@wayfair.com>
  */
+/*eslint-env node */
```

Comment:
It's just how eslint handles overrides, probably works either way.  See http://eslint.org/docs/user-guide/configuring.html.